docs/transpile/defaults_and_configuration_options.mdx

```diff
@@ -0,0 +1,150 @@
+- timing_constraints (Dict[str, int]) –
+- translation_method (str) 
+- unitary_synthesis_method (str)
+- unitary_synthesis_plugin_config (dict)
+
+## Commonly used parameters 
+ 
+### Optimization level
+
+Set the optimization level
+
+Decomposing quantum circuits into the basis gate set of the target device and the addition of swap gates needed to match hardware topology cause an increase the depth and gate count of quantum circuits. To mitigate this increased complexity, you can set the `optimization_level`.  Setting this value calls an optmization routine that optimizes circuits by combining or eliminating gates. In some cases these methods are so effective the output circuits have lower depth than the inputs. In other cases, not much can be done, and the computation may be difficult to perform on noisy devices. 
+
+Different gate optimizations are turned on with different `optimization_level` values. In general, the higher the number, the more optimized it is, at the expense of longer processing time. Choosing the best optimization level might take trial and error, as it depends heavily on the circuit being transpiled and the backend being targeted. By default, `optimization_level=1` is used.
+
+Levels: 
+
+
+The `optimization_level` setting specifies how much optimization to perform on the circuits. Higher levels generate more optimized circuits, at the expense of longer compile times.
+
+- `optimization_level=0`: Trivial optimization, which maps the circuit to the backend with no explicit optimization (except whatever optimizations the mapper does).
+- `optimization_level=1`: Light optimization, which maps the circuit and does simple optimizations by collapsing adjacent gates. This is the default value.
+- `optimization_level=2`: Medium optimization, including a noise-adaptive layout and a gate-cancellation procedure based on gate commutation relationships.
+- `optimization_level=3`: Heavy optimization, which performs the optimization included in levels 1 - 2 and resynthesizes two-qubit gate blocks in the circuit.
+
+<Table>
+  <thead>
+    <Tr>
+      <Th>Optimization Level</Th>
+      <Th>Description</Th>
+    </Tr>
+  </thead>
+  <tbody>
+    <Tr>
+      <Td>0</Td>
+      <Td>
+        No optimization: typically used for hardware characterization
+        - Basic translation
+        - Layout (as specified)
+        - Routing (stochastic swaps)
+      </Td>
+    </Tr>
+    <Tr>
+      <Td>1</Td>
+      <Td>
+        Light optimization (default):
+
+        -   Layout (trivial → vf2 → SabreLayout if routing is required)
+        -   Routing (SabreSWAPs if needed)
+        -   1Q gate optimization
+        -   Error suppression: dynamical decoupling
+      </Td>
+    </Tr>
+    <Tr>
+      <Td>2</Td>
+      <Td>
+        Medium optimization:
+          - Layout/Routing: Optimization level 1 (without trivial) + heuristic optimized with greater
+        search depth and trials of optimization function
+        -   Commutative cancelation
```

Comment:
We also run the 1q gate optimization

Reply:
What about `CXCancellation`?

Reply:
`CXCancellation` only runs at level 1. The optimization it provides is taken care of as part of commutative cancellation. That being said this a good reminder for me that I need to generalize `CXCancellation` because it's not doing anything for backends that don't have CX.